frontend/src/components/meetTheWhales/MeetTheWhalesPage.tsx

```diff
@@ -63,30 +63,33 @@ export function MeetTheWhalesPage(): JSX.Element {
           <li className="whales__list__item" key={i}>
             <div className="whales__card">
               <h2 className="whales__card__title">{w.name}</h2>
-              <h3 className="whales__card__common-name">
-                ({w.species.name} {w.species.latinName})
-              </h3>
-              <figure className="whales__image">
-                <img
-                  className="whales__image"
-                  src={w.photoUrl}
-                  alt={w.name}
-                  width="250px"
-                />
-                <figcaption className="whales__card__info--emphasis">
-                  Conservation status: {w.species.endangeredStatus}
-                </figcaption>
-                <figcaption className="whales__card__info--emphasis">
-                  Description: {w.description}
-                </figcaption>
-                <p className="whales__card__info--sponsors">
-                  Sponsored by our members:
-                  {stringOfUsers(w.users)}
-                </p>
-              </figure>
+              <h5 className="whales__card__common-name">{w.species.name}</h5>
+              <div className="whales__card__grid">
+                <img className="whales__image" src={w.photoUrl} alt={w.name} />
+                <div className="whales__card__info--wrapper">
+                  <p className="whales__card__info--emphasis">
+                    <span className="whales__card__info__title">
+                      Conservation status:
+                    </span>{" "}
```

Comment:
Prettier sometimes likes to add these annoying little `{" "}` things... it's not your fault at all, but please remove them!
```suggestion
                    </span>
```